packages/react-native-platform-override/src/GitReactFileRepository.ts

```diff
@@ -53,9 +53,10 @@ export default class GitReactFileRepository
 
     if (!(await gitClient.checkIsRepo())) {
       await gitClient.init();
+      await gitClient.addConfig('core.filemode', 'false');
+      await gitClient.addConfig('core.autocrlf', 'input');
```

Comment:
This does change behavior, where we default files in the scratch repo to LF line endings. This is actually probably better, since right now we will sometimes get eslint warnings about having CRLF in our new JS until files are staged and gitattributes kicks in.